unit_tests/src/test_atomic_apply_acl_operations.cpp

```diff
@@ -609,9 +609,9 @@ TEST_CASE("#7408: Atomic acls respect entity zone")
     b = std::begin(perms);
     e = std::end(perms);
 
-    // success indicator: should find unique permission "write" with remote zone and shared username
+    // success indicator: should find unique permission "delete_metadata" with remote zone and shared username
     REQUIRE(std::find_if(b, e, [&acl_user_name, &test_zone_name](const fs::entity_permission& p) {
-                return p.name == acl_user_name && p.zone == test_zone_name && p.prms == fs::perms::write;
+                return p.name == acl_user_name && p.zone == test_zone_name && p.prms == fs::perms::delete_metadata;
```

Comment:
Let's avoid changing existing tests if they aren't broken.

Adding a new test will allow you to produce something that focuses on the issue.

Reply:
agreed - consider even making a test that walks through them all(?).

Reply:
new test added... resolving.